common/meta/grpc/src/grpc_client.rs

```diff
@@ -132,7 +165,8 @@ impl MetaGrpcClient {
         let req: Request<RaftRequest> = (&act).try_into()?;
         let req = common_tracing::inject_span_to_tonic_request(req);
 
-        let result = self.client.clone().write_msg(req).await?.into_inner();
+        let mut client = self.make_client_with_tls().await?;
+        let result = client.write_msg(req).await?.into_inner();
```

Comment:
It looks like every time calling a method a new client will be created(with a cached `channel`) and a handshake message will be sent?

There is still chance metasrv restarts just after handshake is well served and before `client.write_msg()` is sent?

Reply:
> It looks like every time calling a method a new client will be created(with a cached channel) and a handshake message will be sent?
This can be changed to using the old token to do action, if encounter an unauthenticated error, fire a handshake request to fetch a new token and replace the old one.  What do you think about this way?

> There is still chance metasrv restarts just after handshake is well served and before client.write_msg() is sent?
Yep, cause `handshake` and `write_msg` are not an atomic operation, so I think this is inevitable. In this case, we need business layers that holds the client to do retries according to their logic.

Reply:
Sorry for the late cause I encounter a deadlock problem:(